lib/tasks/redshift.rake

```diff
@@ -26,7 +26,7 @@ namespace :redshift do
           city character varying, \
           country character varying, \
           email character varying, \
-          in_business boolean, \
+          in_business character varying, \
```

Comment:
👍 

We end up changing the schema often. ... makes me wonder when we should consider automating the drop/create task when it changes.